### PR TITLE
doc: specify _ over - in cloud config modules

### PR DESCRIPTION
**Pull request by powersj**
LP: #1293254

### CODE REVIEW

File: doc/rtd/topics/modules.rst [hunk 1]
```diff
@@ -3,6 +3,10 @@
 *******
 Modules
 *******
+
+This page documents each of the available cloud config modules available in
+cloud-init. Any new modules should use underscores in any new config options.
```
Comment (OddBloke) [inline doc/rtd/topics/modules.rst]: Is this user-facing documentation the appropriate place to record this developer-facing decision? (Would a section in HACKING be more apropos?)

**Comment by OddBloke** [comment]
Not opposed to this landing as-is, but it does seem a little odd.

File: doc/rtd/topics/modules.rst (hunk 1 above)
Comment (powersj) [reply doc/rtd/topics/modules.rst]: I originally went to the HACKING document as well. In its current state it is more about how to get started on cloud-init. However, there are other bugs, like [LP: #1838092](https://bugs.launchpad.net/cloud-init/+bug/1838092), that would be a candidate for putting something in HACKING.

What do you think about starting a "Design" Section in there?

Comment (OddBloke) [reply doc/rtd/topics/modules.rst]: Yeah, that sounds like a great idea!

**Comment by powersj** [comment]
Ok moved to hacking!

**Pull request merged**